packages/block-library/src/comment-template/index.php

```diff
@@ -30,14 +36,21 @@ function block_core_comment_template_render_comments( $comments, $block ) {
 		if ( ! empty( $children ) ) {
 			$inner_content  = block_core_comment_template_render_comments(
 				$children,
-				$block
+				$block,
+				$depth + 1
 			);
 			$block_content .= sprintf( '<ol>%1$s</ol>', $inner_content );
 		}
 
-		$content .= '<li>' . $block_content . '</li>';
+		$comment_classes = comment_class( '', $comment->comment_ID, $comment->comment_post_ID, false );
+
+		$content .= sprintf( '<li %1$s>' . $block_content . '</li>', $comment_classes );
 	}
 
+	// Because $comment_depth is passed as a global, we need to reset before
+	// returning from the function.
+	$comment_depth = $comment_depth - 1;
```

Comment:
> (I'm not sure that subtracting `1` from it would give the desired result -- if it did, we'd have a pretty well-defined relationship between our `$depth` and the `$command_depth` global and would only need to `$depth = $comment_depth + 1;`, no? 🤔 )

Looks like it does 🤔  since the `$depth` is increased by walking down one level.

But this means that `$depth` is really fully determined by `$comment_depth`. So we might as well drop `$depth` 🙂  I'll push a commit!

Reply:
[027f234](https://github.com/WordPress/gutenberg/pull/40330/commits/027f234250a7a62ce1803950e4fc66ba42abae08)